doc/whats_new/v0.20.rst

```diff
@@ -448,6 +448,11 @@ Feature Extraction
   (words or n-grams). :issue:`9147` by :user:`Claes-Fredrik Mannby <mannby>`
   and `Roman Yurchak`_.
 
+- Fixed bug in :class:`feature_extraction.text.TFIDFVectorizer` which 
+  was ignoring the parameter ``dtype``. 
+  :issue:`10441` by :user:`Mayur Kulkarni <maykulkarni>` and
```

Comment:
You don't want to mention `TFIDFTransformer` anymore?

Reply:
ups I messed up.